pkgs/applications/kde/gwenview.nix

```diff
@@ -17,5 +17,5 @@ mkDerivation {
     baloo exiv2 kactivities kdelibs4support kio libkdcraw lcms2 libkipi phonon
     qtimageformats qtsvg qtx11extras
   ];
-  propagatedUserEnvPkgs = [ kipi-plugins ];
+  propagatedUserEnvPkgs = [ kipi-plugins libkipi kinit.bin ];
```

Comment:
Nitpick, but it may be better to use `lib.getBin kinit` because that will always return an output, even when say in the future `kinit` is modified and no longer has a `bin` output.